tasks/eth/008-fp-upgrade/README.md

```diff
@@ -0,0 +1,41 @@
+# Mainnet Fault Proofs Upgrade
```

Comment:
Can generate a report for `input.json` w/ an overview of each transaction w/ `msup render -i input.json -o out.md`, and place it in here. Formatting could use a bit of work, but nice for generating a plain-english overview of each action in the batch.

Reply:
msup is pretty powerful!